PC02_Graffiti_Imhoff.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep  4 11:45:47 2020
+
+@author: hannahimhoff
+"""
+
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: YOUR NAME
```

Comment:
Make sure to add your name to the author line!
```suggestion
Author: Hannah Imhoff
```